### PR TITLE
Add instructions for workshop attendees

### PR DESCRIPTION
**Pull request by eloots**

### CODE REVIEW

File: Hypriot-OS-Course-Preparation-Instructions.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Copy the contents of your *public key* to the `authorized_keys` file:
+
+```
+Pi-Akka-Cluster git:(master) ✗ for node in 0 1 2 3 4;do cat ~/.ssh/id_rsa.pub | ssh akkapi@node-${node} 'cat >> .ssh/authorized_keys'; done
+akkapi@node-0's password:
+```
+
+With this, you should now be able to log into the _akkapi_ on all nodes from the _userxxx_ account on your laptop without having to enter a password.
+
```
Comment (raboof) [inline Hypriot-OS-Course-Preparation-Instructions.md]: I think I've seen this work for people without copying their public key as well - probably their gpg-agent remembered the password after entering it the first time?

:+1: to document this approach - perhaps prefix with "if your ssh does not remember your password, you can optionally do this' to save people who are already set up some work?

Comment (eloots) [reply Hypriot-OS-Course-Preparation-Instructions.md]: That's very interesting! I never used a gpg-agent. It probably also would solve an annoying issue I encounter regularly when working with multiple clusters: when switching between clusters, I need to clean out entries in `~/.ssh/known_hosts`.

Comment (eloots) [reply Hypriot-OS-Course-Preparation-Instructions.md]: @raboof Indeed, that's the plan; actually there's one open PR from @kikiya that aims to do just that. It needs to be adapted a bit as it assumes working with a studentified version of the master repo. This is the long term plan, but right now, we work with the master repo.

Comment (eloots) [reply Hypriot-OS-Course-Preparation-Instructions.md]: @raboof re-phrased instructions based on your comment.
 
> 👍 to document this approach - perhaps prefix with "if your ssh does not remember your password, you can optionally do this' to save people who are already set up some work?

**Pull request merged**